driver/CMakeLists.txt

```diff
@@ -295,19 +301,33 @@ WHILE(${DRIVER_INDEX} LESS ${DRIVERS_COUNT})
 
   MATH(EXPR DRIVER_INDEX "${DRIVER_INDEX} + 1")
 
+  include(FetchContent)
+
+  FetchContent_Declare(
+    json
+    URL https://github.com/nlohmann/json/releases/download/v3.10.5/json.tar.xz
+  )
+
+  FetchContent_Declare(
+    httplib
+    URL https://github.com/yhirose/cpp-httplib/archive/refs/tags/v0.14.0.zip
+  )
+
+  FetchContent_MakeAvailable(httplib json)
+  
   #------------AWS SDK------------------
-  LIST(APPEND SERVICE_LIST rds secretsmanager)
+  LIST(APPEND SERVICE_LIST rds secretsmanager sts)
 
   MESSAGE(STATUS "CMAKE_BUILD_TYPE is ${CMAKE_BUILD_TYPE}")
   IF(MSVC)
     MESSAGE(STATUS "Copying AWS SDK libraries to ${LIBRARY_OUTPUT_PATH}/${CMAKE_BUILD_TYPE}")
     AWSSDK_CPY_DYN_LIBS(SERVICE_LIST "" ${LIBRARY_OUTPUT_PATH}/${CMAKE_BUILD_TYPE})
   ENDIF(MSVC)
 
-  TARGET_INCLUDE_DIRECTORIES(${DRIVER_NAME} PUBLIC ${AWSSDK_INCLUDE_DIR})
-  TARGET_LINK_LIBRARIES(${DRIVER_NAME} ${AWSSDK_LINK_LIBRARIES})
-  TARGET_INCLUDE_DIRECTORIES(${DRIVER_NAME_STATIC} PUBLIC ${AWSSDK_INCLUDE_DIR})
-  TARGET_LINK_LIBRARIES(${DRIVER_NAME_STATIC} ${AWSSDK_LINK_LIBRARIES})
+  TARGET_INCLUDE_DIRECTORIES(${DRIVER_NAME} PUBLIC "${httplib_SOURCE_DIR}" ${AWSSDK_INCLUDE_DIR} ${OPENSSL_INCLUDE_DIR})
+  TARGET_LINK_LIBRARIES(${DRIVER_NAME} nlohmann_json::nlohmann_json ${AWSSDK_LINK_LIBRARIES})
+  TARGET_INCLUDE_DIRECTORIES(${DRIVER_NAME_STATIC} PUBLIC "${httplib_SOURCE_DIR}" ${AWSSDK_INCLUDE_DIR} ${OPENSSL_INCLUDE_DIR})
+  TARGET_LINK_LIBRARIES(${DRIVER_NAME_STATIC} nlohmann_json::nlohmann_json ${AWSSDK_LINK_LIBRARIES})
```

Comment:
Should we move the `json` libraries to it's own? Doesn't seem to belong with `AWS SDK` as the comment suggest